### PR TITLE
Fix continuous processing on errors.

### PR DESCRIPTION
**Pull request by djanowski**

### CODE REVIEW

File: lib/queues.js [hunk 1]
```diff
@@ -342,7 +342,8 @@ class Queue {
             this._activeJobs += jobs.length;
             jobs.forEach(job => {
               this._runAndDestroy(client, job)
-                .then(() => --this._activeJobs);
+                .then(() => --this._activeJobs)
+                .catch(() => --this._activeJobs);
```
Comment (assaf) [inline lib/queues.js]: How about `catch( noop ).then( --this._activeJobs )`?

**Comment by assaf** [comment]
There should have been a test for that, apparently not a very reliable test, since it should have but didn't fail: https://github.com/assaf/ironium/blob/master/test/processing/continuously_test.js#L100

**Comment by djanowski** [comment]
> There should have been a test for that, apparently not a very reliable test, since it should have but didn't fail: https://github.com/assaf/ironium/blob/master/test/processing/continuously_test.js#L100

True, needed to disable concurrency for it to fail: 8db780a682841612910e2849473be5e81c29b097

File: lib/queues.js (hunk 1 above)
Comment (djanowski) [reply lib/queues.js]: Much better, thanks.

**Comment by djanowski** [comment]
@assaf Ready?

**Review by assaf — APPROVED** [review]

**Pull request merged**